.github/workflows/daily_scraper.yml

```diff
@@ -0,0 +1,64 @@
+# Daily fetch bipm website
+
+name: Daily Scrapper
+
+on:
+  schedule:
+    # Runs at UTC 1 am every day
+    # - cron: '0 1 * * *'
+    # Run at every 10 mintues for test
+    - cron: '10 * * * *'
+    
+jobs:
+  scrap:
+
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout outcomes
+      uses: actions/checkout@v3
+      with:
+        ref: daily-scrapping # This line is for test purpose and must be removed later.
+        path: bipm-data-outcomes
+      
+    - name: Checkout tools repo
```

Comment:
We should use the released bipm-data-importer gem instead of from the main Git branch.

This code should just be `gem install bipm-data-importer` and then execute its commands.